scripts/mesos-agent-setup.ps1

```diff
@@ -83,6 +83,7 @@ function New-MesosWindowsAgent {
                            " --isolation=`"windows/cpu,filesystem/windows`"" + `
                            " --containerizers=`"docker,mesos`"" + `
                            " --attributes=`"${mesosAttributes}`"" + `
+                           " --executor_registration_timeout=10mins" + `
```

Comment:
Changes look good.
However, I am wondering how 10 mins is pick? 
There is a agent_reregister_timeout in the mesos  master, this value also happens to be 10 mins.
Would this change intersect with it at all?

Reply:
agent recovery timeout is 15 mins. Would it make sense to push executor_register_timeout to be closer to that number to increase chance to complete downloading  docker images on Windows?

Reply:
I made it configurable via the script variables, because it should be configured by the user to serve his needs (I also added a comment from the documentation).
I also bumped it to 15 minutes.
Regarding the actual value it's more about how much time would you want to wait in case you have to pull down the docker image. In the case of the CI it waited 1 minute instead of 2/3 triggering an error.
I chose the initial 10 minutes because it was used in the unit tests:
https://github.com/apache/mesos/blob/0dcf5c7f0a013f67fff1fb012d1f45455f394708/src/tests/default_executor_tests.cpp#L2577-L2579
Let me know what you think about the updated PR.
Thanks,
Alin.